### DISCUSSION TITLE
Hatch

### DISCUSSION THREAD

**Discussion by ludaavics** [obendidi/httpx-cache#75] [General]
Probably not the best place to ask this but... I've noticed that you switched from poetry to hatch build system.... Would you mind sharing what was the catalyst?

Thanks!

**Comment by obendidi** [comment]
No fancy reason really, 

The major feature of poetry is dependency resolution, which is less useful in libraries that in a webapp or a deployed service, since in a library the deployed artifact is the wheel package that doesn;t contain fully resolved deps (and shouldn't!)

Hatch is the new kid in town, this project seemed like a good candidate to test it out^^
